demo/demo/settings/base.py

```diff
@@ -237,3 +186,6 @@ def get_env_var(setting, warn_only=False):
         value = value.replace('"', '')  # remove start/end quotes
 
     return value
+
+SLACK_TOKEN = get_env_var("SLACK_TOKEN")
+SLACK_CLIENT = SlackClient(SLACK_TOKEN)
```

Comment:
I wonder if there's a way to avoid having to define this here?  It would ideal if it was defined _inside_ response, so people don't need to add this to their settings file.

Reply:
Yes, I've since been wondering this myself actually. We could create a global `slack_client` object in `response.slack.client` that is instantiated using `settings.SLACK_TOKEN`. I'll put that in this afternoon 👍

Reply:
I had a go at this. It's excruciatingly difficult to mock, because of the way Python imports things - I'm running into lots of the issues described in http://alexmarandon.com/articles/python_mock_gotchas/, and looks like we'd need to monkeypatch every single file that `slack_client` would be imported into.

Definitely agree that this isn't a great approach, but I'm not sure it's something we should spend lots of time solving right now